tests/robot-tests/tests/admin/bau/create_data_block_with_chart.robot

```diff
@@ -337,6 +335,26 @@ Configure basic line chart
 
     user clicks link    Data sets
     user waits until h3 is visible    Data sets
+
+    user chooses select option    id:chartDataSetsConfigurationForm-location    Nailsea Youngwood
+    user clicks button    Add data set
+
+    user chooses select option    id:chartDataSetsConfigurationForm-location    Syon
+    user clicks button    Add data set
+
+    user chooses select option    id:chartDataSetsConfigurationForm-location    Bolton 001
+    user clicks button    Add data set
+
+    user checks chart legend item contains    id:chartBuilderPreview    1    Admission Numbers (Nailsea Youngwood)
+    user checks chart legend item contains    id:chartBuilderPreview    2    Admission Numbers (Syon)
+    user checks chart legend item contains    id:chartBuilderPreview    3    Admission Numbers (Bolton 001)
+
+Change data sets
+    user clicks link    Data sets
+
+    user clicks button    Remove all
+    user clicks button    Confirm
+
```

Comment:
This is a bit messy as the test cases should test specific things, but we're jamming a few things into `Configure basic line chart`.

Would suggest we organise the tests in the following way instead:

```robot
Configure basic line chart options
    user clicks button    Line
    user clicks radio    Set an alternative title
    user enters text into element    id:chartConfigurationForm-title    Test chart title
    user enters text into element    id:chartConfigurationForm-alt    Test chart alt
    user enters text into element    id:chartConfigurationForm-height    400
    user enters text into element    id:chartConfigurationForm-width    900

Validate changing data sets
    user clicks link    Data sets
    user waits until h3 is visible    Data sets
  
    user chooses select option    id:chartDataSetsConfigurationForm-location    Nailsea Youngwood
    user clicks button    Add data set
  
    user chooses select option    id:chartDataSetsConfigurationForm-location    Syon
    user clicks button    Add data set
  
    user chooses select option    id:chartDataSetsConfigurationForm-location    Bolton 001
    user clicks button    Add data set
  
    user checks chart legend item contains    id:chartBuilderPreview    1    Admission Numbers (Nailsea Youngwood)
    user checks chart legend item contains    id:chartBuilderPreview    2    Admission Numbers (Syon)
    user checks chart legend item contains    id:chartBuilderPreview    3    Admission Numbers (Bolton 001)

    user clicks button    Remove all
    user clicks button    Confirm

    # Note that we need some tests to check that the chart legend has been updated correctly as well!

Configure line chart data sets
    user chooses select option    id:chartDataSetsConfigurationForm-location    Nailsea Youngwood
    user clicks button    Add data set

    ...
```